packages/config/readme.md

```diff
@@ -1,4 +1,126 @@
-
 # Hops Config
 
-T.B.D. (To Be Documented)
+hops-config exposes options to configure the other Hops packages and Webpack.
+
+# Installation
+When you install [hops-cli](https://www.npmjs.com/package/hops-cli), hops-config will be installed as a dependency as well - so normally, you don't have to separately install hops-config (and we don't recommend that). If you for some reason need to do that anyways, just do:
```

Comment:
I think we _do_ recommend installing `hops-config` manually iff you want to import/require it in downstream code.